### ISSUE TITLE
Parallel commit testing plan

### ISSUE THREAD

**Issue by nrc** [tikv/sig-transaction#23]
Beyond simple unit tests, we should have a plan to test parallel commit thoroughly. Some ideas:

* Running our various test suites in CI with parallel commit enabled.
* Benchmarking to ensure performance
* Adding parallel commit to our TLA+ model

More ideas (and elaborating on the above) welcome!

**Comment by youjiali1995** [comment]
I will introduce it to QA team and they will design the test plan for it.

**Comment by cfzjywxk** [comment]
@nrc @MyonKeminta  
Do we need to list specific development tasks in issues?  We could learn which task is assigned or available, and we could track the progress. Some tasks with high priorities could be labeled and we could make it run if they are done.

**Comment by nrc** [comment]
> Do we need to list specific development tasks in issues

Do you mean for the test plan, or more generally?

There is https://github.com/tikv/tikv/issues/8126, https://github.com/tikv/tikv/issues/8127, https://github.com/tikv/tikv/issues/8128 for the latter, which gets us to the end of the first iteration. I haven't filed TiDB issues for that side, but have some notes locally.

**Comment by youjiali1995** [comment]
I will introduce it to @zyguan next week.

**Comment by cfzjywxk** [comment]
Got, I've found this https://github.com/tikv/tikv/projects/34#card-40571966

**Comment by cfzjywxk** [comment]
> I will introduce it to @zyguan next week.

Yes, this feature is more complex, we need to get @zyguan involved and start to design tests earlier

**Comment by zyguan** [comment]
Sorry for the late reply, I'm working on it.

**Comment by nrc** [comment]
> Sorry for the late reply, I'm working on it.

No problem! Is there anywhere we can track work? Is there anything we can help with?

**Comment by nrc** [comment]
https://docs.google.com/document/d/18T_3UsJ6DFOmurJ8dmuDuDz9acWktKnuv8r9Lj20f7Q/edit?ts=5f07ca1f
